prql-compiler/prqlc/src/cli.rs

```diff
@@ -67,8 +67,21 @@ enum Command {
         format: Format,
     },
 
+    /// Parse, resolve, lower into RQ & preprocess SRQ
+    #[command(name = "sql:preprocess")]
```

Comment:
Traditionally I guess this would be a subcommand `prqlc sql preprocess`, but totally fine as-is